api/.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea
 /playlist/.idea
 .DS_Store
```

Comment:
Move .DS_Store and .idea to a root .gitignore please. And keep everything else on this ignore file that’s related to the api project, please.